common/djangoapps/external_auth/tests/test_ssl.py

```diff
@@ -77,186 +77,6 @@ def setUp(self):
         self.mock = Mock()
 
     @unittest.skipUnless(settings.ROOT_URLCONF == 'lms.urls', 'Test only valid in lms')
-    def test_ssl_login_with_signup_lms(self):
-        """
-        Validate that an SSL login creates an eamap user and
-        redirects them to the signup page.
-        """
-
-        response = external_auth.views.ssl_login(self._create_ssl_request('/'))
-
-        # Response should contain template for signup form, eamap should have user, and internal
-        # auth should not have a user
-        self.assertIn('<form role="form" id="register-form" method="post"', response.content)
-        try:
-            ExternalAuthMap.objects.get(external_id=self.USER_EMAIL)
-        except ExternalAuthMap.DoesNotExist, ex:
-            self.fail('User did not get properly added to external auth map, exception was {0}'.format(str(ex)))
-
-        with self.assertRaises(User.DoesNotExist):
-            User.objects.get(email=self.USER_EMAIL)
-
-    @unittest.skipUnless(settings.ROOT_URLCONF == 'cms.urls', 'Test only valid in cms')
-    def test_ssl_login_with_signup_cms(self):
```

Comment:
Please revisit these test cases. I don't think you should be removing either of these CMS test cases.

You're removing the old LMS login and registration pages in this PR, but you're not touching any CMS (e.g., Studio) pages. Relatedly, code used by the CMS login page should not be removed.